test/e2e/scheduling/taints.go

```diff
@@ -19,20 +19,18 @@ package scheduling
 import (
 	"time"
 
+	. "github.com/onsi/ginkgo"
+	_ "github.com/stretchr/testify/assert"
+	"k8s.io/api/core/v1"
```

Comment:
Don't we need to put a blank line between `_ "github.com/stretchr/testify/assert"` and `"k8s.io/api/core/v1"` like test/e2e/scheduling/priorities.go ?